scripts/generate.ts

```diff
@@ -62,12 +72,24 @@ async function main({ clean }: { clean: boolean }) {
   const rosOutDir = path.join(repoRoot, "ros_foxglove_msgs");
   const typescriptTypesDir = path.join(repoRoot, "typescript/schemas/src/types");
 
+  const pythonSdkRoot = path.resolve(repoRoot, "python", "foxglove-sdk");
+  const pythonSdkGeneratedRoot = path.join(pythonSdkRoot, "src", "generated");
+  const pythonSdkStub = path.join(
+    pythonSdkRoot,
+    "python",
+    "foxglove",
+    "_foxglove_py",
+    "schemas.pyi",
+  );
```

Comment:
fyi with `path.join` you can just use `/` directory separators, it will work cross platform (also I doubt this generate script will work on windows anyway, with all the subcommands it calls)

Reply:
```suggestion
  const pythonSdkRoot = path.resolve(repoRoot, "python/foxglove-sdk");
  const pythonSdkGeneratedRoot = path.join(pythonSdkRoot, "src/generated");
  const pythonSdkStub = path.join(pythonSdkRoot, "python/foxglove/_foxglove_py/schemas.pyi");
```

Reply:
Thanks, I pushed that change after seeing your comment but before the suggestion here.